julia-storage/Dockerfile

```
@@ -0,0 +1,12 @@
+# Ref: https://github.com/tuna/tunasync-scripts/tree/master/dockerfiles/julia
+FROM ustcmirror/base:alpine-3.11
+
+ENV JULIA_DEPOT_PATH="/opt/julia/depot"
+RUN apk update && apk add --no-cache python3 gnupg && \
```

```suggestion
RUN apk update && apk add --no-cache curl python3 gnupg && \
```

`0.2.0` 将下载工作交给 `curl` 在做，所以需要补充一个依赖